### DISCUSSION TITLE
Red node icon for invalid input

### DISCUSSION THREAD

**Discussion by gielians** [VentuzTechnology/Ventuz.Extension.prerelease#23] [Q&A]
Is there a way to force another (red) icon for invalid inputs like the expression nodes do?
![image](https://user-images.githubusercontent.com/20226862/197803448-007c06e0-7327-4f01-81d3-efafd39f8503.png)

**Comment by VentuzDanielWiller** [comment]
You can implement the interface [IIconIndex](https://github.com/VentuzTechnology/Ventuz.Extension.prerelease/wiki/Documentation#icon-index) to change your Node Icon.
Check out the [String Compare sample](https://github.com/VentuzTechnology/Ventuz.Extension.prerelease/blob/main/snippets/SimpleCompare.cs)
